apps/ledger-live-mobile/src/screens/PostOnboarding/PostOnboardingHub.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { Box, Flex, Icons, Log, Notification, Text } from "@ledgerhq/native-ui";
+import { useTranslation } from "react-i18next";
+import { ScrollView, StyleSheet } from "react-native";
+import { SafeAreaView } from "react-native-safe-area-context";
+import styled from "styled-components/native";
+import { useFocusEffect, useNavigation } from "@react-navigation/native";
+import Animated, {
+  interpolate,
+  runOnJS,
+  useAnimatedStyle,
+  useSharedValue,
+  withDelay,
+  withTiming,
+} from "react-native-reanimated";
+import { StackScreenProps } from "@react-navigation/stack";
+import {
+  useAllPostOnboardingActionsCompleted,
+  useClearLastActionCompletedCallback,
+  usePostOnboardingHubState,
+} from "../../logic/postOnboarding/hooks";
+import PostOnboardingActionRow from "../../components/PostOnboarding/PostOnboardingActionRow";
+import { NavigatorName } from "../../const";
+
+const SafeContainer = styled(SafeAreaView).attrs({
+  edges: ["left", "bottom", "right"],
+})`
+  flex: 1;
+`;
+
+const Divider = styled(Box).attrs({
+  height: 0.5,
+  backgroundColor: "neutral.c30",
+  width: "100%",
+})``;
```

Comment:
@gsoares-ledger added a divider component on a separate pr, if you are introducing new components add them to the library instead of inline here or we start to go back to having many copies of components that do basically the same all over the place.